src/styles/stories/Colors/Colors.stories.mdx

```diff
@@ -1,57 +1,65 @@
 import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs';
 import '../../scss/colors.scss';
 
-<Meta title="Colors" />
+import colors from './colors.module.scss';
+
```

Comment:
I thought we weren't using css modules, because it would cause problems with bundling/rendering... Will it not?